Sources/Fuzzilli/CodeGen/CodeGeneratorWeights.swift

```diff
@@ -91,6 +91,7 @@ public let codeGeneratorWeights = [
     "ConstructorGenerator":                     5,
     "SubroutineReturnGenerator":                3,
     "YieldGenerator":                           2,
```

Comment:
While you're here, can you bump these weights to maybe 5 each? These generators will only run when we're inside a generator function, but then we probably do want to run them with a decent probability, so 2 seems a bit low (some day we might find a better way to assign these weights than manual estimates...)

Reply:
Sure！